docs/cow-protocol/reference/core/auctions/bonding_pools.md

```diff
@@ -0,0 +1,19 @@
+---
+id: bonding-pools
+---
+
+# Bonding pools
+
+A bonding pool is an Ethereum mainnet safe whose sole owner is the CoW DAO, and which contains $0.5M worth of (yield-bearing) stable coins and 1.5M COW, as specified by [CIP-7](https://snapshot.org/#/cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7). When a solver wants to join the solver competition, it first needs to join a bonding pool. The main rationale behind this is that a solver in the live competition gets access to the settlement contract, its buffers and the orderflow of the protocol, which means that the attack vector is non-trivial. For the protocol to be protected from a potentially malicious solver, a substantial bonding pool is required so that in case of misbehavior, there are sufficient funds to reimburse the affected parties (users and/or the DAO).
+
```

Comment:
Can we have explicit subsections on
1. How to create a bonding pool (with example transactions)
2. How to join a bonding pool (the vouching part)
3. How to dissolve a bonding pool (the whole project blanc spiel)